src/2-2_Compute-metrics_doublets.r

```diff
@@ -0,0 +1,46 @@
+################################################
+## 2-2_Compute-metrics_scrublet.r
+##  - Compute doublet scores per sample
+##  - Save an txt file with doublet scores per sample [doublet-scores-sampleName]
+################################################
+
+suppressWarnings(library(Seurat))
+suppressWarnings(library(scDblFinder))
+
+message("reloading old matrices...")
+scdata_list <- readRDS("/output/pre-doublet-scdata_list.rds")
+
+message("Loading configuration...")
+config <- RJSONIO::fromJSON("/input/meta.json")
+
+# Check which samples have been selected. Otherwiser we are going to use all of them. 
+if (length(config$samples)>0){
+    samples <- config$samples
+}else{
+    samples <- names(scdata_list)
+}
+
+scdata_list <- scdata_list[samples]
+
+# compute_doublet_scores function
+#' @description Save the result of doublets scores per sample. 
+#' @param scdata Raw sparse matrix with the counts for one sample.
+#' @param sample_name Name of the sample that we are preparing.
+#'
+#' @return 
+compute_doublet_scores <- function(scdata, sample_name, min.features = 10) {
+  
+    message("Sample --> ", sample_name, "...")
+    scdata_DS <- scDblFinder(scdata[, Matrix::colSums(scdata>0)>=min.features])
```

Comment:
```suggestion
    scdata_DS <- scDblFinder(scdata[, Matrix::colSums(scdata>0)>=min.features], dbr = NULL, trajectoryMode = FALSE)
```
I feel we should make some important parameters explicit since we might want to be able to change them in the future. Feel free to ignore when you disagree :)

Reply:
Happy with it!